certified-connectors/Impexium/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
```

Comment:
@RjImpx The connectionParameters property has mentioned with empty body in apiProiperties.json file. Empty connection object will be created automatically without any input from an user. It is less secure to use empty connection parameters. We usually suggest partners to have right connection. Please let us know if you want keep no authentication for your connector.

Reply:
Hello @sriyen-msft , we want to keep no authentication for our connector for now. Thanks